### ISSUE TITLE
Add Formatting for Blog Posts

### ISSUE THREAD

**Issue by nathanjslim** [Hack-Weekly/onyx-sal-blog#58]

**Comment by nathanjslim** [comment]
Current blog posts do not have formatting enabled (or featured). No new lines, line breaks, indents, etc. 
Perhaps linked to issue #45 or can be addressed when we implement the feature to create posts?

**Comment by StephenODea54** [comment]
@nathanjslim agreed with linking to issue #45 . I'm going to rewrite it so it's more up to date with what we need.
